frontend/packages/helm-plugin/src/catalog/utils/catalog-utils.tsx

```diff
@@ -43,7 +43,7 @@ export const normalizeHelmCharts = (
         const imgUrl = chart.icon || getImageForIconClass('icon-helm');
         const chartURL = chart.urls[0];
         const encodedChartURL = encodeURIComponent(chartURL);
-        const href = `/catalog/helm-install?chartName=${name}&chartRepoName=${chartRepoName}&chartURL=${encodedChartURL}&preselected-ns=${activeNamespace}`;
+        const href = `/catalog/helm-install?chartName=${name}&chartRepoName=${chartRepoName}&chartURL=${encodedChartURL}&preselected-ns=${activeNamespace}&indexEntry=${key}`;
```

Comment:
Interessting that we encode chartURL here, but not in our API calls. And here we should also escape name and chartRepoName, right? Or are both kubernetes resources names? It might the safest way to encode them anyway.